variables.tf

```diff
@@ -128,3 +128,17 @@ variable "bucket_versioning" {
   description = "Enable bucket versioning"
   default     = true
 }
+
+variable "origin" {
+  type        = list(object)
+  description = "API gateway origins blocks"
```

Comment:
I'd call this just "Pass through origin blocks", it's not necessarily API Gateway. Same below.